index.ts

```diff
@@ -2,7 +2,6 @@ export { NOOP } from './src/constants/noop';
 export { TConstructible } from './src/types/constructible';
 export { DeepPartial } from './src/types/deep-partial';
 export { Diff } from './src/types/diff';
-export { Omit } from './src/types/omit';
```

Comment:
Is now built into Typescript, so we don't need it anymore.

Reply:
❤️